examples/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "ng serve --open",
```

Comment:
personally most of the time this auto-launch annoys me, since every time you restart it keeps opening new tabs
And to open it you can just ctrl+click on link which is provided in console